google-ads-examples/src/main/java/com/google/ads/googleads/examples/errorhandling/HandleKeywordPolicyViolations.java

```diff
@@ -0,0 +1,214 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.ads.googleads.examples.errorhandling;
+
+import com.beust.jcommander.Parameter;
+import com.google.ads.googleads.examples.utils.ArgumentNames;
+import com.google.ads.googleads.examples.utils.CodeSampleParams;
+import com.google.ads.googleads.lib.GoogleAdsClient;
+import com.google.ads.googleads.v3.common.KeywordInfo;
+import com.google.ads.googleads.v3.common.PolicyViolationKey;
+import com.google.ads.googleads.v3.enums.AdGroupCriterionStatusEnum.AdGroupCriterionStatus;
+import com.google.ads.googleads.v3.enums.KeywordMatchTypeEnum.KeywordMatchType;
+import com.google.ads.googleads.v3.errors.GoogleAdsError;
+import com.google.ads.googleads.v3.errors.GoogleAdsException;
+import com.google.ads.googleads.v3.errors.PolicyViolationDetails;
+import com.google.ads.googleads.v3.resources.AdGroupCriterion;
+import com.google.ads.googleads.v3.services.AdGroupCriterionOperation;
+import com.google.ads.googleads.v3.services.AdGroupCriterionServiceClient;
+import com.google.ads.googleads.v3.services.MutateAdGroupCriteriaResponse;
+import com.google.ads.googleads.v3.utils.ResourceNames;
+import com.google.protobuf.StringValue;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Demonstrates how to request an exemption for policy violations of a keyword.
+ *
+ * <p>The example uses an exemptible policy-violating keyword by default. If you use a keyword that
+ * contains non-exemptible policy violations, they will not be sent for exemption request and you
+ * will still fail to create a keyword.
+ *
+ * <p>If you specify a keyword that doesn't violate any policies, this example will just add the
+ * keyword as usual, similar to what the AddKeywords example does.
+ *
+ * <p>Once you've requested policy exemption for a keyword, when you send a request for adding it
```

Comment:
Suggested reword: When you send a request to add a keyword after requesting a policy exemption for that keyword, the request will pass as if you were adding a non-violating keyword.